submission-folder/TiwKangXu.ts

```diff
@@ -0,0 +1,132 @@
+/**
+ * HOMEWORK
+ * rename this file to *your name*.ts
+ * Translate this JS code into valid TypeScript that follows good practices,
+ * following instructions in the comments
+ */
+
+// replace all `var` declarations appropriatly
+
+let NINE_THOUSAND = 9000
+let number = 6
+number = 7
+
+// create a type alias (rmb type alias names use PascalCase)
+type Student = {
+    name: string
+    year: number
+    friends: string[]
+    favouriteVariable: string
```

Comment:
you are suppose to assign a union type to `favouriteVariable` such that it can be either string or number